File: VacationTrackingSoftware/VacationTrackingSoftware/ClientApp/src/app/InterfacesAndClasses/UserVacationRequest.ts
import { StatusesRequest } from '../Enums/StatusesRequest';

export interface UserVacationRequest {
  id: number;
  userName: string;
  userId: string;
  startDate: Date;
  endDate: Date;
  vacationType: string;
  status: StatusesRequest;
  payment: number;
}  

File: VacationTrackingSoftware/BLL/Models/UserVacationRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace BLL.Models
{
   public class UserVacationRequest
    {
        public int Id { get; set; }
        //public Worker Worker { get; set; }
        public AppUser User { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public VacationType VacationType { get; set; }
        public int Payment { get; set; }
        public int Status { get; set; }
    }
}
